src/stream/src/executor/managed_state/dynamic_filter.rs

```diff
@@ -73,7 +74,7 @@ impl<S: StateStore> RangeCache<S> {
     pub fn insert(&mut self, k: ScalarImpl, v: Row) -> StreamExecutorResult<()> {
```

Comment:
later we should change the Row to RowRef to reduce unnecessary copy

Reply:
Actually, we need to insert concrete `Row` datatype into statetable. So I think passing Row is the right option.